packages/tx/src/transactionFactory.ts

```diff
@@ -110,4 +110,18 @@ export class TransactionFactory {
     const normedTx = normalizeTxParams(txData)
     return TransactionFactory.fromTxData(normedTx, txOptions)
   }
+
+  /**
+   * Method to decode data retrieved from RPC, such as `eth_getTransactionByHash`
+   * Note that this normalizes some of the parameters
+   * @param txData The RPC-encoded data
+   * @param txOptions The transaction options
+   * @returns
+   */
+  public static async fromRPCTx(
```

Comment:
Should we consider reusing this logic inside `fromEthersProvider`?  They're doing basically the same thing except `fromEthersProvider` does an additional fetch call to actually retrieve the data.

Reply:
Yup, great idea